Below the `lg` breakpoint, drop out of the horizontal layout for the scrolling record list.

drgrice1 opened the pull request:
This is one idea.

drgrice1 committed:
Between the `md` and `lg` breakpoint, drop out of the horizontal layout
for the scrolling record list.

Alex-Jordan commented:
I haven't looked at this, but I suspect I would not want this. As much as possible, keeping the two selects sidebyside is helpful to keep my brain in order here. Scrolling up and down a lot with similar looking chunks is not my favorite. But I will load this and look after I get out of a meeting I'm about to head into.

drgrice1 commented:
This doesn't make the selects go vertical (as it does after you drop below the `md` breakpoint), it just makes the sort, format, and filter labels go on top of their respective inputs instead of to the left.

drgrice1 committed:
Add `font-sm` to the union/intersect checkbox labels.

drgrice1 committed:
Also add a small tweak to make the selects on the Answer Log page closer
in size.  The "Users" scrolling select is now a bit longer due to the
addition of the intersect/union radio buttons.
